packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/flash/FlashFeature.java

```diff
@@ -4,35 +4,40 @@
 
 package io.flutter.plugins.camera.features.flash;
 
+import android.annotation.SuppressLint;
 import android.hardware.camera2.CaptureRequest;
+import androidx.annotation.NonNull;
 import io.flutter.plugins.camera.CameraProperties;
 import io.flutter.plugins.camera.features.CameraFeature;
 
 /** Controls the flash configuration on the {@link android.hardware.camera2} API. */
 public class FlashFeature extends CameraFeature<FlashMode> {
-  private FlashMode currentSetting = FlashMode.auto;
+  @NonNull private FlashMode currentSetting = FlashMode.auto;
 
   /**
    * Creates a new instance of the {@link FlashFeature}.
    *
    * @param cameraProperties Collection of characteristics for the current camera device.
    */
-  public FlashFeature(CameraProperties cameraProperties) {
+  public FlashFeature(@NonNull CameraProperties cameraProperties) {
     super(cameraProperties);
   }
 
+  @NonNull
   @Override
   public String getDebugName() {
     return "FlashFeature";
   }
 
+  @SuppressLint("KotlinPropertyAccess")
```

Comment:
Why suppress, rather than changing the field to default access?

Reply:
By "default access", do you mean to remove the `public` modifier? If so, the `CameraFeature.java` file is in a separate directory of `FlashFeature.java`, so it has to stay public to be overrideable. 

The error:
```
/Users/bmparr/Development/packages/packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/flash/FlashFeature.java:34: Error: The getter return type (FlashMode) is not the same as the super return type (T); they should have exactly the same type to allow value be accessed as a property from Kotlin; see https://android.github.io/kotlin-guides/interop.html#property-prefixes [KotlinPropertyAccess]
  public FlashMode getValue() {
```

Reply:
Oh, I wasn't reading carefully enough. I thought this was a `SythenticAccessor` violation that you were suppressing. Ignore me!